R/f7-inputs.R

```diff
@@ -957,21 +957,28 @@ createInputLayout <- function(
   listify(item)
 }
 
-listify <- function(tag, style = NULL) {
-  is_wrapped <- sum(
+#' Check if f7List is in call stack
+#' @keywords internal
+is_wrapped <- function() {
```

Comment:
@DivadNojnarg could be used in more places to determine whether or not the tag is in a list